### PR TITLE
Ippudo locations

### PR DESCRIPTION
**Pull request by codeinabox**

### CODE REVIEW

**Comment by codeinabox** [comment]
@LaoshuBaby could you please offer some guidance on how to treat the [Asian locations according to Wikipedia](https://en.wikipedia.org/wiki/Ippudo#Locations), as I am not sure which ones would use the name Ippudo and which would use 一風堂

File: data/brands/amenity/restaurant.json [hunk 1]
```diff
@@ -6727,7 +6738,7 @@
       "displayName": "一風堂",
       "id": "ippudo-4eae31",
       "locationSet": {
-        "include": ["cn", "jp", "tw"]
+        "include": ["cn", "hk", "jp", "tw"]
```
Comment (tommylung) [inline data/brands/amenity/restaurant.json:6741]: CN included HK.

**Comment by tommylung** [comment]
> @LaoshuBaby could you please offer some guidance on how to treat the [Asian locations according to Wikipedia](https://en.wikipedia.org/wiki/Ippudo#Locations), as I am not sure which ones would use the name Ippudo and which would use 一風堂

Normally, China (including Hong Kong and Macao, but some cases would like to use bilingual for HK & MO) would like to use Simplified Chinese. Taiwan would like to use Traditional Chinese. Other Asian country can use English by default.

**Comment by bhousel** [comment]
Thansks @codeinabox & @tommylung  👍

**Comment by codeinabox** [comment]
> Thansks @codeinabox & @tommylung 👍

Thank you for merging it in. I had meant to squash merge, but not the end of the world.

**Pull request merged**